File: module-3-java/spring-boot-tut/src/main/java/com/example/demo/model/UserPayApp.java
package com.example.demo.model;

import javax.persistence.*;

import lombok.Data;

/**
 * The type User pay app.
 */
@Entity
@Data
@Table(name = "user_pay_app")
public class UserPayApp {
	@Id
	@GeneratedValue
	private int id;

	@Column(name = "user_id")
	private String userId;

	@Column(name = "app_id")
	private int appId;

	@Column(name = "total_amount")
	private double totalAmount;

}

File: module-3-java/bi-tool/pages/components/userpayapp.js
import MUIDataTable from 'mui-datatables'

const columns = [
	{
		name: 'userId',
		label: 'User ID',
		options: {
			filter: false,
			sort: false
		}
	},
	{
		name: 'appId',
		label: 'App ID',
		options: {
			filter: false,
			sort: false
		}
	},
	{
		name: 'totalAmount',
		label: 'Total Amount',
		options: {
			sort: true
		}
	}
]

const options = {
	download: false,
	print: false,
	isRowSelectable: false
}

const UserPayApp = ({ data }) => {
	return (
		<MUIDataTable title={'User Pay App'} data={data} columns={columns} options={options} className="w-2/3 my-2" />
	)
}

export default UserPayApp
